x-pack/plugins/ml/public/application/components/loading_indicator/loading_indicator.tsx

```diff
@@ -13,7 +13,14 @@ export const LoadingIndicator: FC<{ height?: number; label?: string }> = ({ heig
   height = height ? +height : 100;
   return (
     <div
-      className="ml-loading-indicator"
+      css={{
+        textAlign: 'center',
+        fontSize: '17px',
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+      }}
       style={{ height: `${height}px` }}
```

Comment:
Shall we use `EuiFlexGroup` component instead? and use a variable from euiTheme instead of hardcoded 17px?

Reply:
Updated in https://github.com/elastic/kibana/pull/152240/commits/c0b3e32ad44dbd2646b2c2e627c3b9858d3c812b